airbyte-server/src/main/java/io/airbyte/server/services/AirbyteGithubStore.java

```diff
@@ -26,13 +27,12 @@
 public class AirbyteGithubStore {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(AirbyteGithubStore.class);
-
+  private static final EnvConfigs envConfigs = new EnvConfigs();
   private static final String GITHUB_BASE_URL = "https://raw.githubusercontent.com";
   private static final String SOURCE_DEFINITION_LIST_LOCATION_PATH =
-      "/airbytehq/airbyte/master/airbyte-config/init/src/main/resources/seed/source_definitions.yaml";
+      "/airbytehq/airbyte/" + envConfigs.getGithubStoreBranch() + "/airbyte-config/init/src/main/resources/seed/source_definitions.yaml";
```

Comment:
What do you think about expanding the env var? 
You could make the default value be: 
`/airbytehq/airbyte/master/airbyte-config/init/src/main/resources/seed/`

It would allow OSS users to set a custom github store on their own repo.

Reply:
I like it, but that seems like a specific feature we should think more about (are there security implications?  Do we want that in the UI?)... so it should be in another PR at least :D